packages/elements/src/photon-multirx-form/components/DraftPrescriptionCard.tsx

```diff
@@ -33,18 +24,14 @@ export const DraftPrescriptionCard = (props: {
   const [editDialogOpen, setEditDialogOpen] = createSignal<boolean>(false);
   const [editDraft, setEditDraft] = createSignal<any>(undefined);
   const [deleteDraftId, setDeleteDraftId] = createSignal<string | undefined>();
+  const { prescriptionIds, setEditingPrescription, deletePrescription } = usePrescribe();
 
-  props.actions.registerValidator({
-    key: 'draftPrescriptions',
-    validator: draftPrescriptionsValidator
-  });
-
-  const dispatchPrescriptionDraftDeleted = (prescription: DraftPrescription) => {
+  const dispatchPrescriptionDraftDeleted = (id: string) => {
     const event = new CustomEvent('photon-draft-prescription-deleted', {
       composed: true,
       bubbles: true,
       detail: {
-        prescription: unwrap(prescription)
+        prescriptionId: id
```

Comment:
this is probably fine, this technically breaks our contract since there might be customers who listen on this ... but I doubt it